misc: Added colors enum to constants

kyranet opened the pull request:
Though it also changes some colors, for example, message delete logs will be red instead of orange, as requested by user feedback.

kyranet committed:
misc: Added colors enum to constants

favna reviewed (commented):
Slotmachine also has some color constants, maybe replace those too?

other than that, LGTM.

kyranet committed:
fix: Resolved ME#splitFields bug

kyranet committed:
fix: Use DeepPurple in followage too

kyranet committed:
fix: Change mute from DeepOrange to Amber

It was harder to distinguish between Red (Leave) and Muted Member Join (DeepOrange), so I changed this to amber.
